.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 dist
 .DS_Store
 .idea
+packages/example/contracts
```

Comment:
Noise?

Reply:
Those are files generated with typechain 

> "typechain:generate": "yarn typechain --target=ethers-v5 ./src/abi/**/*.json --out-dir=./contracts"

Propably better not to include them in repo